packages/core/components/QueryPart/QueryPartRow.module.css

```diff
@@ -27,16 +30,16 @@
 
```

Comment:
Seeing a background box around the up down arrows. Is this intentional? 
![image](https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/94479316/2b439e81-d63d-45e1-91cd-1d591117dff1)

Reply:
ummmm idk why there are even up and down arrows for you... what browser is this?

Reply:
chrome!

Reply:
weird hopefully just a Windows thing... I'm going to try to remove with an overflow attribute but I'll add an issue to do some extensive browser and os testing